README.md

```diff
@@ -45,7 +45,7 @@ iex ((New-Object System.Net.WebClient).DownloadString('https://install.mondoo.co
 Install-Mondoo;
 ```
 
-If you prefer manual installation, you find the cnquery packages on  [GitHub releases](https://github.com/mondoohq/cnquery/releases).
+If you prefer manual installation, you can find the cnquery packages in our [releases](https://github.com/mondoohq/cnquery/releases).
```

Comment:
Any reason not to call it out as being our github releases page?

Reply:
Not really. I was changing awkward language and in the process I just left it out for brevity. Is there value in pointing out that it's GH?

Reply:
We have a lot of different places to get the files so it's a bit nice to be clear what we're talking about this time

Reply:
Cool beans.